contracts/token/ERC777/ERC777.sol

```diff
@@ -176,7 +176,7 @@ contract ERC777 is Context, IERC777, IERC20 {
     /**
      * @dev See {IERC777-isOperatorFor}.
      */
-    function isOperatorFor(address operator, address tokenHolder) public view override returns (bool) {
+    function isOperatorFor(address operator, address tokenHolder) public view virtual override returns (bool) {
```

Comment:
For this one and `defaultOperators`... I can imagine people might want to override this with a more efficient implementation that doesn't use storage, or something. But ERC777 specifies that the default operators should never change, so this is something people would have to know if they override them. I guess they shouldn't expect to be able to override a function specified in the ERC without checking that their semantics are compliant with the spec.

Reply:
Not sure what you are trying to say. Keeping it virtual with a reminder of the expected behaviour & ERC specifications ? Not having it virtual ?

Reply:
Sorry I rambled a bit. I was unsure but I think if it can be easily made virtual in a consistent way then we should do it, and people should be familiar with the spec requirements if they ever override it.